app/notifications/rest.py

```diff
@@ -97,7 +97,7 @@ def send_notification(notification_type):
     if errors:
         raise InvalidRequest(errors, status_code=400)
 
-    check_rate_limiting(authenticated_service, api_user)
+    check_rate_limiting(authenticated_service, api_user, notification_type)
```

Comment:
👍 this makes sense to me, i like this pattern of passing the notification type since we will now have 2 different checks